xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-index/pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>xwiki-platform-flavor-api</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>us.springett</groupId>
+      <artifactId>cvss-calculator</artifactId>
+      <version>1.4.1</version>
```

Comment:
@tmortagne do we have a rule for putting a dep with version directly inside main platform-core pom to simplify future upgrades / discovery of the dependencies we use?